src/effects/builtin/reverb/ReverbView.qml

```diff
@@ -1,20 +1,104 @@
 import QtQuick 2.15
 
+import Muse.UiComponents
+
 import Audacity.Effects
 
 import "../common"
 
 EffectBase {
 
-    property string title: reverb.title
+
     property alias instanceId: reverb.instanceId
 
-    width: 300
-    height: 200
+    property string title: qsTrc("effects", "Reverb")
+    property bool isApplyAllowed: true
+
+    height: 400
+    width: 400
 
     ReverbViewModel {
         id: reverb
     }
 
+    Component.onCompleted: {
+        reverb.init()
+    }
+
+    component ParamItem: Item {
+        id: param
+        property alias title: label.text
+        property real value: 0.0
+        property alias min: slider.from
+        property alias max: slider.to
 
+        height: 40
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.margins: 8
+
+        StyledTextLabel {
+            id: label
+            anchors.left: parent.left
+            anchors.top: parent.top
+            anchors.bottom: parent.bottom
+            verticalAlignment: Text.AlignVCenter
+            horizontalAlignment: Text.AlignLeft
+            width: 120
+        }
+
+        IncrementalPropertyControl {
+            id: numField
+            anchors.left: label.right
+            anchors.verticalCenter: parent.verticalCenter
+            width: 60
+            currentValue: param.value
+            onValueEdited: function(newVal) {
+                param.value = newVal
+            }
+        }
+
+        StyledSlider {
+            id: slider
+            anchors.right: parent.right
+            anchors.left: numField.right
+            anchors.leftMargin: 8
+            anchors.verticalCenter: parent.verticalCenter
+            stepSize: 0.1
+            value: param.value
+            onMoved: param.value = slider.value
+        }
+    }
+
+    Column {
+        anchors.top: parent.top
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.bottom: wetOnly.top
```

Comment:
It looks strange not to let the `CheckBox` specify `anchors.top: theColumn.bottom`. Would that have worked, too? (You don't have to change if it does, it's more out of curiosity.)

Reply:
Here's the logic: 
* this checkbox should be stuck to the bottom, not to the parameters (i think)
* to stick to a column, it must have geometry, i.e. a height is set. For a column, this is simple, but for example in a similar case for ListView with scrolling, when the number of elements can be different and even more than the height of the dialog, then this will all be a problem

Reply:
Aha!